tests/framework/utils/k8s_helper.go

```diff
@@ -64,6 +63,29 @@ const (
 	hostnameTestPrefix = "test-prefix-this-is-a-very-long-hostname-"
 )
 
+func TestPlatform() string {
```

Comment:
Should this func be in env.go since it's based on an env var?

Reply:
Done.